checkstyle-suppressions.xml

```diff
@@ -4,13 +4,8 @@
      "http://www.puppycrawl.com/dtds/suppressions_1_0.dtd">
 
 <suppressions>
-
-  <suppress checks="LineLength" files="com/google/api/client/googleapis/apache/GoogleApacheHttpTransport.java"/>
-  <suppress checks="LineLength" files="com/google/api/client/googleapis/extensions/appengine/notifications/AppEngineNotificationServlet.java"/>
-  <suppress checks="LineLength" files="com/google/api/client/googleapis/extensions/appengine/subscriptions/package-info.java"/>
-  <suppress checks="LineLength" files="com/google/api/client/googleapis/extensions/servlet/notifications/NotificationServlet.java"/>
-  <suppress checks="LineLength" files="com/google/api/client/googleapis/javanet/GoogleNetHttpTransport.java"/>
-  <suppress checks="LineLength" files="com/google/api/client/googleapis/subscriptions/NotificationHeaders.java"/>
+  <suppress checks="LineLength" files="com[/\\]google[/\\]api[/\\]client[/\\]googleapis[/\\]extensions[/\\]appengine[/\\]notifications[/\\]AppEngineNotificationServlet.java"/>
+  <suppress checks="LineLength" files="com[/\\]google[/\\]api[/\\]client[/\\]googleapis[/\\]extensions[/\\]servlet[/\\]notifications[/\\]NotificationServlet.java"/>
```

Comment:
I had to add forward slash and back slash to make it work for both Windows and Unix filesystem.